snorkel/parser/parser.py

```diff
@@ -18,15 +24,15 @@ def to_unicode(self, text):
         :param text:
         :return:
         '''
-        text = text.encode('utf-8', 'error')
-        # Python 2
+        # Python 2 only -- should fail on LookupError in Python 3
```

Comment:
Perhaps make the logic more explicit:

```
if sys.version_info[0] < 3:
```

Don't allow the python 2 steps to be run on python 3?